testsuite/cluster-test/src/experiments/performance_benchmark_three_region_simulation.rs

```diff
@@ -54,9 +54,12 @@ impl Experiment for PerformanceBenchmarkThreeRegionSimulation {
         );
         join_all(network_effects.iter().map(|e| e.activate())).await;
         let window = Duration::from_secs(240);
-        let emit_job_request = EmitJobRequest {
-            instances: self.cluster.validator_instances().to_vec(),
-            ..context.global_emit_job_request.clone()
+        let emit_job_request = match context.global_emit_job_request {
```

Comment:
This seem to be repeating pattern, should it be just function in EmitJobRequest?
Smth like `EmitJobRequest::with_default(Option<..>, Vec<Instance)`